src/seedu/addressbook/data/person/Name.java

```diff
@@ -40,7 +40,9 @@ public static boolean isValidName(String test) {
      * Retrieves a listing of every word in the name, in order.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        List<String> wordsInName = Arrays.asList(fullName.split("\\s+"));
+        wordsInName.replaceAll(String::toLowerCase);
+        return wordsInName;
```

Comment:
This implementation removes the functionality of retrieving words in name in original case. Should this be done?
Further, you should update the header comment of this method to reflect that it returns words in lower case.